libs/website/builder/src/lib/website-builder.context.tsx

```diff
@@ -143,6 +144,7 @@ export type WebsiteBuilderProps = {
   elements: {
     Rating: ComponentType<BuilderRatingProps>
     Alert: ComponentType<BuilderAlertProps>
+    ApolloErrorAlert: ComponentType<BuilderApolloAlertProps>
```

Comment:
Had to include the component into Website Builder. Otherwise a circular dependency would happen.